dev-lang/jimtcl/files/jimtcl-0.78-jimsh0-cflags.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/autosetup/autosetup-find-tclsh b/autosetup/autosetup-find-tclsh
```

Comment:
Could you please merge it with the previous patch? It's a bit fragile to apply two separate patches to the same file, and both are Gentoo-specific.